industrial_ci/ros_pre-release.sh

```diff
@@ -93,3 +95,16 @@ function run_ros_prerelease() {
 
     return $RESULT_PRERELEASE
 }
+
+function run_ros_prerelease() {
+	retval_prerelease=-1
+	if [ ! "$PRERELEASE_OS_CODENAME" ]; then
```

Comment:
PRERELEASE_OS_CODENAME was meant to override the default, e.g. to test another OS variant